adagrams/game.py

```diff
@@ -1,13 +1,117 @@
 from random import randint
 
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    
+    pool_word = []
+    hand = []
+    for char,quant in LETTER_POOL.items():
+        
+        pool_word.extend([char] * quant)
+        
+    
+    for i in range(10):
+        ind = randint(0,len(pool_word)-1)
```

Comment:
Spacing issues: Assignment, arithmetic, and comparison operators should have a blank space on either side to make it easier for other developers to read and understand our code as quickly and easily as possible. 

```suggestion
        ind = randint(0, len(pool_word) - 1)
```